core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -1277,7 +1277,8 @@ public static boolean isRemoteSegmentWithinLeaderEpochs(RemoteLogSegmentMetadata
      * does not contain any messages/records associated with them.
      *
      * For ex:
-     *  <epoch - start offset>
+     * <pre>
+     *  &lt;epoch - start offset&gt;
```

Comment:
Does it make sense to cover this and other 'tables' in `@code` like:

```java
* <pre>
* {@code
* <epoch - start offset>
* ...
* }
* </pre>
```

to allow for readability even when reading the class file directly?

Reply:
Addressed in [01e23fb](https://github.com/apache/kafka/pull/15527/commits/01e23fb2520338bcd49a69da478fea37859de00a)